client/src/components/ConcludedIssue.js

```diff
@@ -33,6 +33,16 @@ class ConcludedIssue extends React.Component {
     };
   }
 
+  componentWillReceiveProps(nextProps) {
+    this.setState({
+      majority: ConcludedIssue.calculateMajority(
```

Comment:
I know this isn't really a part of this PR, but IMO this should not be a state. 
A state that can be derived from other states/props is not needed.

I think the whole component can be coverted to a pure component and majority can be calculated when rendering.